Source/Krypton Toolkit/Examples/CircularProgressBar/Form1.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Krypton.Toolkit;
```

Comment:
Sorting ?

Reply:
My installation was sorting the statements, only found out after checking this in. Will solve this.

Reply:
Need the System.### ones first. (It's a setting in the Visual Studio Options)